scripts/rm_all.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# Removes all files named all.lean in the src/ directory
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+find $DIR/../src/ -name 'all.lean' -delete
```

Comment:
Shouldn't you also remove the `olean` files? Seems nice to clean up those as well.

Reply:
I'll do that!